src/ray/core_worker/core_worker.cc

```diff
@@ -1861,14 +1871,18 @@ void CoreWorker::SubmitActorTask(const ActorID &actor_id, const RayFunction &fun
                              ? function.GetFunctionDescriptor()->DefaultTaskName()
                              : task_options.name;
   const std::unordered_map<std::string, std::string> override_environment_variables = {};
+  int64_t depth;
+  {
+    absl::MutexLock lock(&mutex_);
+    depth = current_task_.GetDepth();
```

Comment:
Why not +1?

Reply:
added comment, but it doesn't really matter since depth is inherited from its actor creation task